client/src/services/translation.ts

```diff
@@ -53,6 +54,46 @@ export function getWorkspaceRoleTranslationKey(role: WorkspaceRole | null): Work
   }
 }
 
+interface CustomOrderStatusTranslations {
+  title: Translatable;
+  description?: Translatable;
+}
+
+export function getCustomOrderStatusTranslationKey(status: CustomOrderStatus | undefined): CustomOrderStatusTranslations {
+  switch (status) {
+    case undefined: {
+      return {
+        title: 'clientArea.dashboard.processing.error.title',
+      };
+    }
+    case CustomOrderStatus.NothingLinked:
+      return {
+        title: 'clientArea.dashboard.processing.requestSent.title',
+        description: 'clientArea.dashboard.processing.requestSent.description',
+      };
+    case CustomOrderStatus.EstimateLinked:
+      return {
+        title: 'clientArea.dashboard.processing.estimateLinked.title',
+        description: 'clientArea.dashboard.processing.estimateLinked.description',
+      };
+    case CustomOrderStatus.InvoiceToBePaid:
+      return {
+        title: 'clientArea.dashboard.processing.invoiceToBePaid.title',
+        description: 'clientArea.dashboard.processing.invoiceToBePaid.description',
+      };
+    case CustomOrderStatus.InvoicePaid:
+      return {
+        title: 'clientArea.dashboard.processing.organizationAvailable.title',
+        description: 'clientArea.dashboard.processing.organizationAvailable.description',
+      };
+    default:
+      return {
+        title: 'clientArea.dashboard.processing.error.title',
+        description: 'clientArea.dashboard.processing.error.description',
+      };
+  }
+}
+
```

Comment:
store the prefix `clientArea.dashboard.processing` in a variable to avoid repetition and improve readibility.